website/docs/d/netapp_volume.html.markdown

```diff
@@ -0,0 +1,78 @@
+---
+subcategory: "NetApp"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_netapp_volume"
+sidebar_current: "docs-azurerm-datasource-netapp-volume"
+description: |-
+Gets information about an existing NetApp Volume
+---
+
+# Data Source: azurerm_netapp_volume
+
+Uses this data source to access information about an existing NetApp Volume.
+
+
+## NetApp Volume Usage
+
+```hcl
+data "azurerm_netapp_volume" "example" {
+  resource_group_name = "acctestRG"
+  account_name        = "acctestnetappaccount"
+  pool_name           = "acctestnetapppool"
+  name                = "acctestnetappvolume"
```

Comment:
can we use more readable naming here e.g.

```suggestion
  name                = "example-volume"
```

Reply:
Fixed